cpp/ql/src/Security/CWE/CWE-704/WcharCharConversion.qhelp

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+  <p>This rule indicates a potentially incorrect cast from/to an ANSI string (<code>char *</code>) to/from a Unicode string (<code>wchar_t *</code>).</p>
+  <p>This cast might yield strings that are not correctly terminated; including potential buffer overruns when using such strings with some dangerous APIs.</p>
+</overview>
+
+<recommendation>
+  <p>Do not explicitly casting ANSI strings to/from Unicode strings.</p>
```

Comment:
Ideally we should give a recommendation that can make the alert go away and keep the code working. What is the typical remedy for this sort of error? For string literals we can recommend prepending `L`. For other cases, should we recommend calling an appropriate (platform-dependent) conversion function?